src/GeositeFramework/plugins/layer_selector_v2/draw_report/main.js

```diff
@@ -58,6 +60,16 @@ define(["dojo/_base/declare",
                 this.bindEvents();
             },
 
+            // Translate the built-in draw tool text
+            setupDrawingText: function() {
+                var drawText = esriText.toolbars.draw,
+                    props = ['start', 'resume', 'complete'];
+
+                _.each(props, function(prop) {
+                    drawText[prop] = i18next.t(drawText[prop]);
```

Comment:
I think we should be using abbreviations for i18n keys. If we ever upgrade to a newer version of the JSAPI and these text values change, our translation files will silently break.